### PR TITLE
Allow fullscreen lecture units for more browsers

### LINKED ISSUE #3267: full screen for lecture-unit videos not working with some browsers
### Describe the bug
Some browsers don't allow lecture unit videos to go full screen. This problem occurs for both, youtube and tum-live videos.

#### To Reproduce

0. Use Chromium (in my case. 90.0.4430.72)
1. Open a video unit (YouTube or TUM-Live)
2. Full screen button is visible but disabled.

#### Expected behavior
Full screen buttons should be clickable like they are in Firefox for example
#### Screenshots

Chromium (not working) | Firefox (working)
--- | ---
image | image

### Environment

<details><pre>
 - OS: Kubuntu 20.10
 - Browser Chromium
 - Version 90.0.4430.72
</pre></details>

#### Output in Web Console

<details><pre>
  Nothing interesting
</pre></details>

### PR DESCRIPTION
**Pull request by joschahenningsen**
### Checklist
- [x] I tested *all* changes and *all* related features with different users (student, tutor, instructor, admin) on the test server https://artemistest.ase.in.tum.de.
- [x] Client: I followed the [coding and design guidelines](https://artemis-platform.readthedocs.io/en/latest/dev/guidelines/client.html).

### Motivation and Context

fixes #3267

### Description
strict feature policy header was blocking fullscreen iframes for some chromium based browsers. Honestly seems like a bug to me because the iframe should be able to override this but loosening the feature policy does the trick.

### Steps for Testing

0. Find a browser that doesn't allow full screen lecture units on the current Artemis instance (Chrome should be a good fit. In my case it's Chromium v 90.0.4430.85 (snap)). If you can't find a browser that doesn't work you won't be able to review this :(
1. Create a video unit (try a youtube video and a TUM-Live video, e.g. `https://live.mm.rbg.tum.de/w/eist/991?video_only=1`)
2. Check if fullscreen works now.
3. Check if fullscreen still works in other browsers.

### CODE REVIEW

**Comment by joschahenningsen** [comment]
There seems to be some other obscure reason for fullscreen being blocked. I can't fully wrap my head around it yet. 
Breadcrumb: strangely the full screen mode works just fine when the webapp is accessed via port 9060. 8080 does not work.

**Comment by jpbernius** [comment]
Can this be related to a `Feature-Policy` header?

**Comment by joschahenningsen** [comment]
Possible, I'll take another look at it tomorrow.

**Comment by joschahenningsen** [comment]
Unfortunately setting a fullscreen: * header doesn't help.

File: src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html [hunk 1]
```diff
@@ -14,7 +14,7 @@ <h5 class="card-header unit-card-header" [class.rounded]="isCollapsed" (click)="
         </h5>
         <div class="card-body unit-card-body" [ngbCollapse]="isCollapsed">
             <div class="embed-responsive embed-responsive-16by9">
-                <iframe id="videoFrame" class="embed-responsive-item rounded" [src]="videoUrl" allow="fullscreen" allowfullscreen></iframe>
+                <iframe id="videoFrame" class="embed-responsive-item rounded" [src]="videoUrl" allow="fullscreen" allowfullscreen="1"></iframe>
```
Comment (GeneralMine) [inline src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html:17]: have you tried to add the following: `allowFullScreen="true" webkitallowfullscreen="true" mozallowfullscreen="true"` ?
You could also playaround with the `frameborder="0"` property.

**Review by GeneralMine — CHANGES_REQUESTED** [review]
try this, could work..

File: src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html (hunk 1 above)
Comment (joschahenningsen) [reply src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html]: It was the feature policy after all. When testing it I just missted a `'` :facepalm:

**Review by GeneralMine — APPROVED** [review]
tested on ts3 using Firefox and Chrome. Works for me ✅

File: src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html (hunk 1 above)
Comment (GeneralMine) [reply src/main/webapp/app/overview/course-lectures/video-unit/video-unit.component.html]: happens.. but well done 😀😉👍

**Pull request merged**